Google.Api.Gax.Grpc.Gcp/GcpCallInvokerPool.cs

```diff
@@ -37,7 +37,7 @@ public sealed class GcpCallInvokerPool
         /// </summary>
         /// <param name="scopes">The scopes to apply. Must not be null, and must not contain null references. May be empty.</param>
         public GcpCallInvokerPool(IEnumerable<string> scopes) =>
-            _credentialsCache = new DefaultChannelCredentialsCache(scopes);
```

Comment:
Are you going to add another constructor here? At the moment `DefaultChannelCredentialsCache` will never use JWTs.

Reply:
If I understood it correctly, it is not used by GAPIC libraries and might be removed soon, therefore, setting to default. Let me know if you think it is not a good idea.

Reply:
It's used by the Spanner and Bigtable libraries. If we don't mind those APIs not getting self-signed JWTs, then let's revert everything within this project. There seems to be little point in implementing the functionality but not actually allowing it to be used.

Reply:
thanks for the clarification, i'll add a constructor so Spanner and BigTable can use it

Reply:
added